setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 88
-max-complexity = 6
+max-complexity = 8
```

Comment:
I am curious about this change. did the tests not pick it up before?

Reply:
The problem was `if "SENTRY_DSN" in env and not is_in_shell:` was considered too complex - which just seems silly

Reply:
perhaps that line should get a `# noqa: C901` ? 🤔

Reply:
Perhaps, but then also i'd argue that line is definitely at a level of complexity which is fine in a codebase, hence changing the rule rather than special-casing that specific line.